linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/router_server.js

```diff
@@ -0,0 +1,28 @@
+var RouterServer = (function() {
+  // Skeletal implementation to demonstrate functionality of RouterController
+
+  function initializeServerContainers(servers, $serverEl) {
```

Comment:
Ah yes! It's a naming convention - if it is a jquery object, I name it $variable.  I don't know if we actually use this convention in this codebase.  Should I change it?